addons/mail/static/src/models/composer/composer.js

```diff
@@ -265,62 +269,70 @@ function factory(dependencies) {
                             threadId: thread.id,
                             threadModel: thread.model,
                         })
                     );
+                    const [messageData] = await this.async(() => this.env.services.rpc({
+                        model: 'mail.message',
+                        method: 'message_format',
+                        args: [[messageId]],
+                    }, { shadow: true }));
+                    this.env.models['mail.message'].insert(Object.assign(
+                        {},
+                        this.env.models['mail.message'].convertData(messageData),
+                        {
+                            originThread: [['insert', {
+                                id: thread.id,
+                                model: thread.model,
+                            }]],
+                        })
+                    );
+                    thread.loadNewMessages();
                 }
-            } else {
-                Object.assign(postData, {
-                    subtype_xmlid: this.isLog ? 'mail.mt_note' : 'mail.mt_comment',
-                });
-                messageId = await this.async(() =>
-                    this.env.models['mail.thread'].performRpcMessagePost({
-                        postData,
-                        threadId: thread.id,
-                        threadModel: thread.model,
-                    })
-                );
-                const [messageData] = await this.async(() => this.env.services.rpc({
-                    model: 'mail.message',
-                    method: 'message_format',
-                    args: [[messageId]],
-                }, { shadow: true }));
-                this.env.models['mail.message'].insert(Object.assign(
-                    {},
-                    this.env.models['mail.message'].convertData(messageData),
-                    {
-                        originThread: [['insert', {
-                            id: thread.id,
-                            model: thread.model,
-                        }]],
-                    })
-                );
-                thread.loadNewMessages();
+                for (const threadView of this.thread.threadViews) {
+                    // Reset auto scroll to be able to see the newly posted message.
+                    threadView.update({ hasAutoScrollOnMessageReceived: true });
+                }
+                thread.refreshFollowers();
+                thread.fetchAndUpdateSuggestedRecipients();
+                this._reset();
+                this.update({ isPostingMessage:false });
             }
-            for (const threadView of this.thread.threadViews) {
-                // Reset auto scroll to be able to see the newly posted message.
-                threadView.update({ hasAutoScrollOnMessageReceived: true });
+            catch(e) {
+                this.update({ isPostingMessage:false });
+                throw e;
             }
```

Comment:
I still need to test myself this magic catch stuff that we somehow don't need to add otherwise around RPC but that has effect when we do have it. Consequently I'm not sure the point of re-throwing, but I guess this could be `finally` instead of `catch` anyway.